aws_cloudtrail.tf

```diff
@@ -6,28 +6,29 @@ resource "aws_cloudtrail" "missing-multi-region" {
   name                          = "tf-trail-foobar"
   s3_bucket_name                = "some-s3-bucket"
   s3_key_prefix                 = "prefix"
-  include_global_service_events = false
+  include_global_service_events = true
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Enable AWS CloudTrail logging for global services**
 
**Vulnerability**
Ensure AWS CloudTrail trails track API calls for global services such as IAM, STS and CloudFront.
 
**Remediation**
Ensure that include_global_service_events is set to true

source: accurics-